pages/en/developer/assemblyscript-migration-guide.mdx

```diff
@@ -91,19 +91,19 @@ maybeValue.aMethod();
 However on the newer version, because the value is nullable, it requires you to check, like this:
 
 ```typescript
-let maybeValue = load()
+let maybeValue = load();
```

Comment:
As mentioned on Slack, please run Prettier (with `yarn prettier`) and you'll see that the semicolons are automatically removed.

Reply:
I've removed all semicolons!